server/converters/cxgtool.py

```diff
@@ -409,6 +412,7 @@ def save_metadata(container, metadata):
     with tiledb.DenseArray(a_name, mode="w") as A:
         A.meta["cxg_version"] = CXG_VERSION
         A.meta["cxg_properties"] = json.dumps(metadata)
+        A.meta["colors"] = json.dumps(anndata_colors_to_cxg_colors(adata))
```

Comment:
please prefix with `cxg_` as this is not a well-defined format  and is therefore unique to us.